### PR TITLE
RN(serverless): Serverless release note

### PR DESCRIPTION
**Pull request by cbehera-newrelic**
- Created a release note section for Serverless monitoring.

### CODE REVIEW

File: src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.11.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+subject: "Lambda-Extension"
+releaseDate: '2024-02-28'
+version: 2.3.11
+---
+
+## Notes
```
Comment (jeff-colucci) [inline src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.11.mdx:7]: This note section is a little confusing, are the crashes being fixed or are we notifying customers that they are happening?

**Review by jeff-colucci — COMMENTED** [review]
This seems largely fine, I did find a few things that need to be fixed. Were these notes existing somewhere else that got transferred over to this site or are they all brand new?

File: src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.12.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+subject: "Lambda-Extension"
+releaseDate: '2024-06-24'
+version: 2.3.12
+---
+
+
+## Notes
+
+* Adds local testing for Extension. Local testing is enabled by setting `NEWRELIC_LOCAL_TESTING` to true. Currently it supports only Python runtime. It generates the lambda layers for `x86` and `arm` architecture layers using `publish.sh` script and invokes the test Lambdas using `test.sh` script.
```
Comment (adutta-newrelic) [inline src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.12.mdx:10]: Can we please add back ticks to true, example `true`

Comment (adutta-newrelic) [reply src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.12.mdx:10]: Suggestion for rephrase:
Adds local testing for Extension. Local testing is enabled by setting `NEWRELIC_LOCAL_TESTING` to true. Currently supports Python runtime only, and generates lambda layers for `x86` and `arm` architecture using `publish.sh` and invokes the test Lambdas using `test.sh` script.

File: src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.13.mdx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Notes
+
+* Introduces support for Ruby Lambda runtime.
+
+* Adds tags for logs, and introduces environment variables to configure custom attributes or other metadata when sending logs directly using extension. Use `NR_TAGS` & `NR_ENV_DELIMITER` environment variables similar to the `log-ingestion-lambda` to add metadata to the logs.
+
+* Removes debug logs which get printed for each log line. Modifies confusing message log - mainLoop: blocking is confusing customer and adds payloadSize to the log message `Telemetry client error: %s, payload size: %d bytes"`.
+
+* Updates on .NET Extension example to use newly released New Relic .NET lambda layer instead of OpenTracing.
```
Comment (adutta-newrelic) [inline src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.13.mdx:16]: Suggestion:
Lamda should be in upper case

**Review by adutta-newrelic — APPROVED** [review]
Hello @cbehera-newrelic I have approved the PR. Thank you.

**Pull request merged**